content/docs/tools/related-tools.md

```diff
@@ -27,6 +27,7 @@ People have written tools on top of Cucumber. These are the known third-party to
 | [Cucumber Growler](https://github.com/paolodona/cucumber_growler)                                               | [Growl](http://growl.info/) notifications for Cucumber.                                                                                                                                                                                                                                      |
 | [Cucumber Nagios](http://auxesis.github.io/cucumber-nagios)                                                     | Web site monitoring with Cucumber and [Nagios](http://www.nagios.org/)                                                                                                                                                                                                                       |
 | [Cucumber Scaffold](https://github.com/andyw8/cucumber_scaffold)                                                | Scaffolding gem for Cucumber Features                                                                                                                                                                                                                                                        |
+| [cucumber-runner](https://plugins.gradle.org/plugin/se.thinkcode.cucumber-runner)                               | A Gradle Cucumber-JVM runner - it may simplify running Cucumber steps in parallel                                                                                                                                                                                                            |
```

Comment:
```suggestion
| [cucumber-runner](https://plugins.gradle.org/plugin/se.thinkcode.cucumber-runner)                               | A Gradle Cucumber-JVM runner which may simplify running Cucumber steps in parallel                                                                                                                                                                                                            |
```

Reply:
Suggested text change, because having ` - ` in this description only looks a bit weird to me.